suite-common/test-utils/src/extraDependenciesMock.ts

```diff
@@ -53,16 +53,23 @@ export const extraDependenciesMock: ExtraDependencies = {
         }),
     },
     actions: {
-        addTransaction: mockAction('addTransaction'),
-        removeTransaction: mockAction('removeTransaction'),
         setAccountLoadedMetadata: mockAction('setAccountLoadedMetadata'),
         setAccountAddMetadata: mockAction('setAccountAddMetadata'),
+        fiatRateUpdate: mockAction('fiatRateUpdate'),
     },
     actionTypes: {
         storageLoad: mockActionType('storageLoad'),
     },
     reducers: {
         storageLoadBlockchain: mockReducer('storageLoadBlockchain'),
         storageLoadAccounts: mockReducer('storageLoadAccounts'),
+        storageLoadTransactions: mockReducer('storageLoadTransactions'),
+    },
+    utils: {
+        // TODO how to mock this
```

Comment:
No need to TODO, this is actually mock. I would suggest just droppning sentence about phone and use just something generic like `Calling mocked "saveAs" to save ${data} into ....` because this mock could be also used in tests etc.

Reply:
I missed the comment -> that was just note to myself and doesn't need to be there anymore. I will delete it.